### ISSUE TITLE
Add support for Python3, please.

### ISSUE THREAD

**Issue by dizballanze** [beeware/cricket#38]
```
$ cricket-django 
Traceback (most recent call last):
  File "/home/nikola/proj/venv/bin/cricket-django", line 9, in <module>
    load_entry_point('cricket==0.2.3', 'console_scripts', 'cricket-django')()
  File "/home/nikola/proj/venv/lib/python3.4/site-packages/pkg_resources.py", line 353, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/nikola/proj/venv/lib/python3.4/site-packages/pkg_resources.py", line 2302, in load_entry_point
    return ep.load()
  File "/home/nikola/proj/venv/lib/python3.4/site-packages/pkg_resources.py", line 2029, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/home/nikola/proj/venv/lib/python3.4/site-packages/cricket/django/__main__.py", line 4, in <module>
    from cricket.main import main as cricket_main
  File "/home/nikola/proj/venv/lib/python3.4/site-packages/cricket/main.py", line 36
    print cricket.VERSION
                ^
SyntaxError: invalid syntax
```

**Comment by chadgh** [comment]
+1

**Comment by eliasdorneles** [comment]
I believe this now can be closed, been using cricket on Python 3.4 and it works fine. :+1:

**Comment by ulgens** [comment]
Hello from 2020, this issue can be closed :slightly_smiling_face:
